### PR TITLE
fix: solidity tests

### PR DESCRIPTION
**Pull request by JordyRo1**
## Fix 

 - Fixes the solidity tests
 - Fixes issue in the solidity contract in the case where there is proof available
 - **TO BE VERIFIED**: Theoros, low <-> high inversion (the data displayed as bytes should be ordered by high then low. Currently it's low then high)

### CODE REVIEW

File: solidity/test/benchmarks/PragmaDecoderGasTest.t.sol [hunk 1]
```diff
@@ -28,7 +28,13 @@ contract PragmaDecoderGasTest is Test {
 
     function testGasAllUpdates() public {
         string[5] memory updateTypes = ["SpotMedian", "TWAP", "RealizedVolatility", "Options", "Perpetuals"];
-        string[5] memory currencies = ["ETH/USD", "BTC/USD", "ETH/USD", "ETH/USD", "ETH/USD"];
+        uint256[5] memory currencies = [
+            uint256(0x4554482f555344),
```
Comment (EvolveArt) [inline solidity/test/benchmarks/PragmaDecoderGasTest.t.sol]: we can define these as constants

File: solidity/test/utils/HyperlaneTestUtils.t.sol [hunk 2]
```diff
@@ -28,10 +28,14 @@ abstract contract HyperlaneTestUtils is Test {
...
         bytes32 emitterAddress,
+        bytes32 merkleTreeHookAddress,
+        bytes32 root,
+        uint32 checkpointIndex,
+        bytes32 messageId,
         bytes memory payload,
         uint8 numSigners
     ) public view returns (bytes memory updateData) {
-        uint8 version = 1;
+        uint8 version = 3;
```
Comment (EvolveArt) [inline solidity/test/utils/HyperlaneTestUtils.t.sol]: same for the version

**Review by EvolveArt — CHANGES_REQUESTED** [review]
nit

File: solidity/test/utils/HyperlaneTestUtils.t.sol (hunk 2 above)
Comment (JordyRo1) [reply solidity/test/utils/HyperlaneTestUtils.t.sol]: Done, will apply further changes in the refacto branch ✅

**Pull request merged**